--- arch/arm/boot/dts/imx6dl-colibri-eval-v3.dts.bak	2017-04-05 14:34:18.449016827 +0800
+++ arch/arm/boot/dts/imx6dl-colibri-eval-v3.dts	2017-05-16 16:00:43.001833099 +0800
@@ -155,6 +155,15 @@
 &hdmi_video {
 	status = "okay";
 };
+/*enable FlexCan1*/
+&flexcan1 {
+        status = "okay";
+};
+
+/*enable FlexCan2*/
+&flexcan2 {
+        status = "okay";
+};
 
 /*
  * I2C: I2C3_SDA/SCL on SODIMM 194/196 (e.g. RTC on carrier
@@ -201,10 +210,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_weim_gpio_1 &pinctrl_weim_gpio_2
 	             &pinctrl_weim_gpio_3 &pinctrl_weim_gpio_4
-	             &pinctrl_weim_gpio_5 &pinctrl_weim_gpio_6
+	             &pinctrl_weim_gpio_5
 	             &pinctrl_csi_gpio_1
 	             &pinctrl_gpio_1
-	             &pinctrl_gpio_2
 	             &pinctrl_usbh_oc_1 &pinctrl_usbc_id_1>;
 
 	gpio {
@@ -260,7 +268,7 @@
 &uart2 {
 	status = "okay";
 #if 0
-	linux,rs485-enabled-at-boot-time;
+       linux,rs485-enabled-at-boot-time;
 #endif
 };
 
